milli/src/update/index_documents/mod.rs

```diff
@@ -436,7 +436,7 @@ impl<'t, 'u, 'i, 'a> IndexDocuments<'t, 'u, 'i, 'a> {
             let readers = rayon::iter::repeatn(documents, num_threads)
                 .enumerate()
                 .map(|(i, documents)| {
-                    let stop_words = fst::Set::default();
+                    let stop_words = self.index.stop_words(self.wtxn)?;
```

Comment:
Could you please fetch the `stop_words` only once outside of this function, for example on line 406, please?